src/components/switch/switch.component.tsx

```diff
@@ -157,8 +164,7 @@ export const Switch = React.forwardRef(
       onChange: isControlled ? onChange : onChangeInternal,
       id,
       name,
-      label,
-      labelHelp,
+      labelHelp: !validationRedesignOptIn ? labelHelp : null,
```

Comment:
**comment** (non-blocking): as far as I can tell the functionality is working fine here, but I find it a bit inconsistent (and was confused at first) how you're handling the `label` prop vs `labelHelp`. In both cases we only want to pass it to `CheckableInput` if `validationRedesignOptIn` is false, but for `labelHelp` you've expressed that with a ternary within `inputProps`, whereas for `label` you've removed it entirely and manually added it as a separate prop in the JSX in the appropriate part of the larger ternary.

I don't have a strong preference for which way is better but the inconsistency bothers me a bit.

Reply:
Good point. I'll pick a way to do both and keep it consistent. I'll pick a way which makes sense for when we remove the old design and the need for the opt in flag.